src/views/DataStream/FormFields/FormFieldsDataStream.vue

```diff
@@ -1341,10 +1341,14 @@
     headers.value.splice(index, 1)
   }
 
-  const insertDataSet = (templateID) => {
+  const insertDataSet = (templateID, isFirstRender) => {
     const index = listTemplates.value.map((el) => el.value).indexOf(templateID)
     try {
-      if (props.resetForm) {
+      if (templateID === 'CUSTOM_TEMPLATE') {
+        if (!isFirstRender) {
```

Comment:
O que acha de combinar isso em um único if? Acho que pode simplificar o código.
```suggestion
      if (templateID === 'CUSTOM_TEMPLATE' && !isFirstRender) {
```